crates/std_lib/src/merkle.nr

```diff
@@ -17,4 +17,4 @@ fn check_membership(_root : Field, _leaf : Field, _depth : Field) -> Field {}
 // and then return the new root
 // The root is only here to check the underlying database. No constraints are applied to the root
 #[foreign(insert_regular_merkle)]
```

Comment:
This `#[foreign(insert_regular_merkle)]` is no longer needed because we don't have this opcode anymore.

`insert` can be implemented by in noir using check_membership similar to how your example program did

Reply:
Done. Perhaps it would be good once simple_shield example is more complete this could a good example in the Noir book. Or even the simple `merkle_insert` example I created as part of this PR.